src/cli/index.ts

```diff
@@ -92,7 +92,7 @@ async function verify(bundlePath: string, artifactPath: string) {
   }
 
   const bundleFile = fs.readFileSync(bundlePath);
-  const bundle = JSON.parse(bundleFile.toString('utf-8'));
+  const bundle = JSON.parse(bundleFile.toString('utf-8')) as sigstore.Bundle;
```

Comment:
@bdehamer ops spotted the ts-reset lib only meant for apps not libs, so would reset the consuming app code 🤦 removed it from this PR.

https://github.com/total-typescript/ts-reset#use-ts-reset-in-applications-not-libraries